stacks/triliovault-operator/upgrade.sh

```diff
@@ -24,8 +28,8 @@ echo "Upgrading Triliovault operator..."
 if [ -z "${MP_KUBERNETES}" ]; then
   # use local version of values.yml
   ROOT_DIR=$(git rev-parse --show-toplevel)
-  values="$ROOT_DIR/stacks/triliovault-operator/values.yml"
-  TVM="$ROOT_DIR/stacks/$STACK/triliovault-manager.yaml"
+  values="$TVK_PATH/values.yml"
+  TVM="$TVK_PATH/triliovault-manager.yaml"
```

Comment:
same as above

Reply:
`$TVK_PATH` parameter is deduced from the `$ROOT_DIR` which would give a absolute path every time and `values.yaml` or `triliovault-manager.yaml` file pat will be generated.